model/project_parser.go

```diff
@@ -1330,9 +1331,11 @@ func evaluateBVTasks(tse *taskSelectorEvaluator, tgse *tagSelectorEvaluator, vse
 			}
 			if err1 != nil && err2 != nil {
 				evalErrs = append(evalErrs, err1, err2)
```

Comment:
No, the error removed (where you suggest a warning- working on that!) is the one that would trigger this line. The warning I'm implementing is at the project validation level, rather than here (project evaluation level).